canonical-websites/netplan.io
# Rewrite examples and add FAQ in markdown

This takes the example page and breaks it out into separate examples and
FAQ topics. The examples are now more focused on specific configurations
and scenarios. It attempts to get the common high-level keys outlined with
a few options.

The FAQ page then is simple questions of how and why things work the way
they do. It also covers questions from users migrating from ifupdown.

Also added a table of contents to both.

This also converts of both pages is in markdown instead of the painful
HTML to aid in user contributions and readability.

Fixes #14
Fixes #19

## Done

* Re-write examples
* Add FAQ
* Added table of contents to both pages
* Switch to markdown
* Explain shadowing of configuration files per #14 

## QA

Used the run.sh script to check and preview my work as I went and verify conversion to markdown.

## Issue / Card

As stated above:
Fixes #14
Fixes #19
